maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dockerImage>${D}{kalixContainerRegistry}/${D}{kalixOrganization}/${D}{project.artifactId}</dockerImage>
     <dockerTag>${project.version}-${build.timestamp}</dockerTag>
     <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
-    <mainClass>${package}.Main</mainClass>
+    <mainClass>${package}.MainKt</mainClass>
```

Comment:
Is this correct? AFAICS, it is `Main` in `Main.kt`.

Reply:
Yeah, Kotlin magic, I've tested this locally.